util/topsql/reporter/reporter.go

```diff
@@ -527,8 +527,7 @@ func (tsr *RemoteTopSQLReporter) getReportData(collected collectedData) reportDa
 		sort.Sort(others)
 	}
 	for _, evict := range evicted {
-		collected.normalizedSQLMap.LoadAndDelete(string(evict.SQLDigest))
-		collected.normalizedPlanMap.LoadAndDelete(string(evict.PlanDigest))
+		// Shouldn't evict the SQL meta, since the evicted SQL can be appear on Other components (TiKV) TopN records.
```

Comment:
This may lead to massive peak network / CPU each minute since the SQL meta capacity is very large.